.github/pull_request_template.md

```diff
@@ -8,6 +8,7 @@
   - in case of new component, add new component page
   - in case of new features, add variation to existing component page
   - nice to have: add it to the demo application as well
+- [ ] do the changes work in dark and light theme?
```

Comment:
To make the checklist consistent I would follow imperative form -> "test changes in both light and dark theme" unless it indicates a list of subtasks, e.g. "is this a breaking change?"